docs/AI-for-security/connect-to-byo.asciidoc

```diff
@@ -0,0 +1,192 @@
+| Quantization reduces overall parameters and helps the model to run faster, but reduces accuracy.
+| Examples: Llama, Mistral, Phi-3, Falcon.
+| The number of parameters is a measure of the size and the complexity of the model. The more parameters a model has, the more data it can process, learn from, generate, and predict.
+| The context window defines how much information the model can process at once. If the number of input tokens exceeds this limit, input gets truncated.
+| Specific formats for quantization vary, most models now support GPU rather than CPU offloading.
+|===
+
+[discrete]
+== Load a model in LM Studio 
+
+After downloading a model, load it in LM Studio using the GUI or LM Studio's https://lmstudio.ai/blog/lms[CLI tool].
+
+[discrete]
+=== Option 1: load a model using the CLI (Recommended)
+
+It is a best practice to download models from the marketplace using the GUI, and then load or unload them using the CLI. The GUI allows you to search for models, whereas the CLI only allows you to import specific paths, but the CLI provides a good interface for loading and unloading.
+
+Use the following commands in your CLI:
+
+1. Verify LM Studio is installed: `lms`
+2. Check LM Studio's status: `lms status`
+3. List all downloaded models: `lms ls`
+4. Load a model: `lms load`
+
+image::images/lms-cli-welcome.png[The CLI interface during execution of initial LM Studio commands]
+
+After the model loads, you should see a `Model loaded successfully` message in the CLI.
+
+image::images/lms-studio-model-loaded-msg.png[The CLI message that appears after a model loads]
+
+To verify which model is loaded, use the `lms ps` command.
+
+image::images/lms-ps-command.png[The CLI message that appears after running lms ps]
+
+If your model uses NVIDIA drivers, you can check the GPU performance with the `sudo nvidia-smi` command.
+
+[discrete]
+=== Option 2: load a model using the GUI
+
+Refer to the following video to see how to load a model using LM Studio's GUI. You can change the **port** setting, which is referenced in the Nginx configuration file. Note that the **GPU offload** was set to **Max**.
+
+=======
+++++
+<script type="text/javascript" async src="https://play.vidyard.com/embed/v4.js"></script>
+<img
+  style="width: 100%; margin: auto; display: block;"
+  class="vidyard-player-embed"
+  src="https://play.vidyard.com/FMx2wxGQhquWPVhGQgjkyM.jpg"
+  data-uuid="FMx2wxGQhquWPVhGQgjkyM"
+  data-v="4"
+  data-type="inline"
+/>
+</br>
+++++
+=======
+
+[discrete]
+== (Optional) Collect logs using Elastic's Custom Logs integration
+
+You can monitor the performance of the host running LM Studio using Elastic's https://www.elastic.co/docs/current/integrations/log[Custom Logs integration]. This can also help with troubleshooting. Note that the default path for LM Studio logs is `/tmp/lmstudio-server-log.txt`, as in the following screenshot:
```

Comment:
```suggestion
You can monitor the performance of the host running LM Studio using Elastic's {integrations-docs}/log[Custom Logs integration]. This can also help with troubleshooting. Note that the default path for LM Studio logs is `/tmp/lmstudio-server-log.txt`, as in the following screenshot:
```